docker-compose-digester.yml

```diff
@@ -0,0 +1,21 @@
+services:
```

Comment:
No good reason to have two separate docker-compose files if using Makefile in the end. Just move it to the main docker-compose file, then use `docker compose up -d tummy` and `docker compose up -d digester`

Reply:
except i am mounting a file in the digester whose path is given by env variable. now if that env variable is not set, even the other services donot start or build. what to do?

Reply:
There are following ways to handle it:
- While running the service which doesn't require something to mount, you can provide the variable with any value (empty value as well) before the ".. docker-compose.." command
- Or specify `env` file for the service which requires it.

Also, afair when not providing env variable it should just give warning. Maybe its specific to `docker-compose` which I use, and you might be using `docker compose`.